sale_order_import_ubl_customer_free_ref/tests/test_ubl_order_import.py

```diff
@@ -0,0 +1,58 @@
+# © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import base64
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
```

Comment:
use Savepoint + tracking_disable... you know :)

Reply:
done